gradle/common-java.gradle

```diff
@@ -126,7 +126,7 @@ if (java7JreDir) {
 } else {
     def msg = "IMPORTANT: Environment variable 'JAVA_JRE_7' is not defined - Install JRE 7 and set 'JAVA_JRE_7' to prevent runtime compatibility issues!"
     if (!isJava7) {
-        if (isBuildServer || isRelease) {
+        if ((isBuildServer && !System.env.SMOKE_APP) || isRelease) {
```

Comment:
I'm not sure I agree with this. If this is still building the artifacts, I still think we need the java 7 jars on the bootstrapclasspath

Reply:
This pipeline is not building the artifacts, just running smoke tests

Reply:
Where does it get the artifacts? I didn't see that part

Reply:
Sorry, what I mean is, this is not replacing the `AppInsights-Java-Gradle-CI` build, only the `AppInsights-Java-SmokeTest-MultiConfig` build.

I don't think we need to compile against Java 7 bootstrap classes when running smoke tests, since any issue will be caught during `AppInsights-Java-Gradle-CI`.